app/views/posts/index.html.erb

```diff
@@ -1,4 +1,22 @@
-<h1>All the posts from a user</h1>
-<h3>Post number 1</h3>
-<h3>Post number 2</h3>
-<h3>Post number 3</h3>
+<% @users = User.all; %>
+<% current_user = @users.find_by(id: params[:user_id]) %>
+<div class="body">
+  <h2>All the posts from <%= current_user.name %></h2>
+  <div>
+    <%= render partial: "users/user", locals: { current_user: current_user } %>
+  </div>
+  <% @posts_per_user = @users.find_by(id: params[:user_id]).posts %>
+  <% @posts_per_user.first(3).each do |post| %>
```

Comment:
- [ ] please avoid using logic in the views, you previously created a method called `last_3_posts` in your user model to handle this.